### DISCUSSION TITLE
Sooooooo all my trains look like crap what am I doing wrong

### DISCUSSION THREAD

**Discussion by TheMindExpansionNetwork** [TheLastBen/fast-stable-diffusion#1929] [General]
I am really trying my best to fine tune the training setup and I noticed that all my models are just not working as I seen lots of others being trained on civitia I cant really use the models for NFTs I think if anyone has insite so I want to be able to train my own concepts.

Is there something that I missing this is kinda what I been using 

20 photos x 200

so 4000 steps on main encode

then 350 for the text.

I am not sure if be honest lets say naked women if I would want to train on style or if I need to really focus on the face.

Does anyone know a good formula I can use that will rock it.

any advice much apricated

**Comment by TheLastBen** [comment]
did you use one single token as filename for all the images ?

**Reply by TheMindExpansionNetwork** [reply]
Thank you for hitting me back up man 

So this is what I am using 

![image](https://user-images.githubusercontent.com/77267159/229401975-d48bed1c-8ff5-4a0e-b140-a7cb26caf3f4.png)


Honestly, I attempting soooooo many it's unreal

so I was thinking this best 

20 images 4000 steps

350 for the text encoder

captions are in blip from processing thing in automatic

Honestly any advice is a good man

I have bunch of clients say girls trying to train them and notice keep just having issues not looking as I need

What you think is good for 20 images

**Reply by TheLastBen** [reply]
don't use captions and use something like "ttkkrkgn" as a token

**Reply by TheMindExpansionNetwork** [reply]
![image](https://user-images.githubusercontent.com/77267159/229403025-55e78297-4bc6-424a-91d8-98150a805172.png)


Does this look a bit better

then what you think for 20 should I keep the basic settings go from there

Firing it up now

**Reply by TheMindExpansionNetwork** [reply]
![image](https://user-images.githubusercontent.com/77267159/229403660-eb8908c4-75d5-40ba-96f0-ec54f3d53b6b.png)


This is my new settings to start

**Reply by TheMindExpansionNetwork** [reply]
![image](https://user-images.githubusercontent.com/77267159/229404509-48d1dd2c-2ef4-4f83-b6e7-8e06fcb49dff.png)

Duuuuuuuude like night and day

So what you think is this something I would want to train more if face is a little off or should it be good to go

I did the settings I provided

SO much better than mine

**Reply by TheMindExpansionNetwork** [reply]
![ttkkrkgn 19](https://user-images.githubusercontent.com/77267159/229404678-bbdd53f2-b626-46e4-954c-9c9d4a29f7c6.png)


this is one I am using

Thank you sooo much for this sorry blowing ya up

**Reply by TheLastBen** [reply]
also, pick 10 instance images instead of 20, use the 10 best

**Reply by TheMindExpansionNetwork** [reply]
Thank you man I am very happy with this I will ensure to do that my next character give you all a update.

Funny how its the little thing I was jacking steps to 4000 at a tune lol and in between

Now it is a lot better. 

10 honestly be better for me

Keep up this amazing work

**Comment by meunumerotim333** [comment]
If you want to use my workflow, feel free to try.

I'm using an old colab, make a copy:
https://colab.research.google.com/drive/1uUUGu5bH1u2RML-XYm01E9ME3KngpZmW#scrollTo=R3SsbIlxw66N

Use maximum 20 pictures and multiplies x100.. the ideal is 10 like Ben said, but 20 depending of variety can be good.. I always focus only on face training like this:

![image](https://user-images.githubusercontent.com/121369033/229405722-fb89a77a-fbaa-4593-87d6-80810a07b2ee.png)

Not all the models will be good!! You need to test them. The deliberate model on hugginface looks cool.

I also train 768x768 pictures. and Focus to create 768x1024 pictures using controlNET or Img2Img to increase the time producing without aberrations.

I don't know why Ben's updates changed the results during the time.. I would like not using old colabs.. I'm a bit insecure if for some reason stop working on the future. But for now its pretty good to me, and hope continues working throught all the updates.

**Reply by meunumerotim333** [reply]
Also use this configuration to improve areas like eyes, hair, mouth using inpaint

![image](https://user-images.githubusercontent.com/121369033/229408109-b75b7f13-e74b-4ea9-867e-f7fe084680da.png)

Play with the denoise strenght, keep the lowest possible when the results is already close to your taste.

**Reply by TheMindExpansionNetwork** [reply]
Thank you all this has been huge help I'm going to be adding to workflow love it all just had that little bump also possible does anyone know good flows for style and knowing when your next model is trained to much
